app/controllers/orders_controller.rb

```diff
@@ -28,11 +38,33 @@ def index
 
   # confirmation page
   def show
-    if params[:view] == 'merchant'
+    if params[:view] == "merchant"
       @order = Order.find(params[:id])
       render :usershow
     else
       @order = Order.find(session[:order_id])
+      @products_info = @order.order_items.map do |item|
+        product = item.product
+        {
+        weight_lbs:  product.weight_lbs,
+        length_in: product.length_in,
+        height_in: product.height_in,
+        width_in: product.width_in,
+        units: product.units,
+        quantity: item.quantity,
+        item_id: item.id
+        }
+      end
+
+      @place = {
+        country: @order.country,
+        state: @order.state,
+        city: @order.city,
+        zip: @order.zip
+      }
+
```

Comment:
Is @place accessed in the view, or is it just used for creating the ShippingWrapper object? If so, it can be a local variable instead of an instance variable.